guides/common/modules/proc_upgrading-the-external-database.adoc

```diff
@@ -1,34 +1,15 @@
 [id="Upgrading_the_External_Database_{context}"]
 = Upgrading the external database
 
-You can upgrade an external database from {EL} 8 to {EL} 9 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
-
-.Prerequisites
-* Create a new {EL} 9 based host for PostgreSQL server that follows the external database on {EL} 9 documentation.
-ifdef::katello,orcharhino,satellite[]
-For more information, see {InstallingServerDocURL}using-external-databases_{project-context}[Using External Databases with {Project}].
-endif::[]
-* Install PostgreSQL version 13 on the new {EL} host.
-
-.Procedure
-. Create a backup.
-. Restore the backup on the new server.
+* If you use an external database, use the `pg_upgrade` utility to upgrade from PostgreSQL 12 to PostgreSQL 13 without upgrading the underlying operating system from {EL} 8 to {EL} 9. 
```

Comment:
I think the method isn't that relevant and in the line below you already give a solution. In the end there's nothing wrong with alternative methods like upgrading the external DB to PostgreSQL 13, but we only give the supported path of in place upgrading.
```suggestion
* If you use an external database, upgrade from PostgreSQL 12 to PostgreSQL 13.
```